src/main/java/com/suse/saltstack/netapi/config/ClientConfig.java

```diff
@@ -12,6 +12,25 @@
     public static final Key<URI> URL = new Key<>(URI.create("http://localhost:8000"));
     public static final Key<String> TOKEN = new Key<>();
 
+    // Request timeout settings
+    /**
+     * Sets a specified timeout value, in milliseconds, to be used when dealing a communications link in HttpClientConnection / JDKConnection.
+     * A timeout of zero is interpreted as an infinite timeout. Default value is 10000ms (10s).
```

Comment:
I would write something more similar to the HttpClient documentation:
`Timeout in milliseconds until a connection is established. A timeout of zero is interpreted as an infinite timeout. Default value is 10000ms (10s)`